pkg/proxy/ipvs/proxier.go

```diff
@@ -157,6 +160,12 @@ type Proxier struct {
 	nodePortSetTCP *IPSet
 	// nodePortSetTCP is the bitmap:port type ipset where stores all UDP node port
 	nodePortSetUDP *IPSet
+	// lbIngressLocalSet is the hash:ip type ipset where stores all service ip which with externaltrafficPolicy=local
+	lbIngressLocalSet *IPSet
+	// nodePortLocalSetTCP is the bitmap:port type ipset where stores all TCP node port with externaltrafficPolicy=local
+	nodePortLocalSetTCP *IPSet
```

Comment:
"nodePortLocalSetTCP is the bitmap:port type ipset which stores all TCP nodeport's with externaltrafficPolicy=local"